File: DataGatewayService/API/Entities/Country.cs
using API.Common;

namespace API.Entities
{
    public class Country : BaseEntity
    {
        public string Name { get; set; } = string.Empty;
        public float CostOfLivingIndex { get; set; } = 0;
        public float RentIndex { get; set; } = 0;
        public float CostOfLivingPlusRentIndex { get; set; } = 0;
        public float GroceriesIndex { get; set; } = 0;
        public float RestaurantPriceIndex { get; set; } = 0;
        public float LocalPurchasingPowerIndex { get; set; } = 0;
        public int Popularity { get; set; } = 0;
    }
}

File: QuickQueryUI/src/app/models/Country.ts
export class CountrySearchResult {
    public id: string | undefined; 
    public name: string | undefined;
    public costOfLivingIndex: number | undefined;
    public rentIndex: number | undefined;
    public costOfLivingPlusRentIndex: number | undefined;
    public groceriesIndex: number | undefined;
    public restaurantPriceIndex: number | undefined;
    public localPurchasingPowerIndex: number | undefined;
    public createdAt: Date | undefined;
    public updatedAt: Date | undefined;
    public popularity: number | undefined;
}
